src/main/java/com/nha/abdm/wrapper/hip/HIPClient.java

```diff
@@ -89,4 +98,35 @@ public ResponseEntity<HealthInformationBundleResponse> healthInformationBundleRe
         .toEntity(HealthInformationBundleResponse.class)
         .block();
   }
+
+  public ResponseEntity<ProfileAcknowledgement> shareProfile(
+      ShareProfileRequest shareProfileRequest) {
+    return webClient
+        .post()
+        .uri(shareProfilePath)
+        .accept(MediaType.APPLICATION_JSON)
+        .body(BodyInserters.fromValue(shareProfileRequest))
+        .retrieve()
+        .toEntity(ProfileAcknowledgement.class)
+        .block();
+  }
+  public <T> ResponseEntity<ResponseOtp> fetchResponseFromHIPForOtp(String uri, T request) {
```

Comment:
since this class is HIPClient, we can simply rename the methods to requestOtp and not use `fetchResponseFromHIP`

Reply:
Sure modifying it into requestOtp.